x11-misc/revelation/revelation-0.5.3.ebuild

```diff
@@ -0,0 +1,50 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Do you need to use *gnome2.eclass*? I know it sets some defaults for you, but you should be fine using simple *autotools.eclass* here.

You can use the latest EAPI-7 that we prefer if you get rid of outdated *gnome2.eclass*

Reply:
without gnome2.eclass gnome2_src_install wont work so I stick to EAPI 6 for the time being